src/clusterfuzz/_internal/tests/appengine/handlers/external_update_test.py

```diff
@@ -118,6 +121,7 @@ def test_update_changed_security(self):
     self.testcase.put()
 
     stacktrace = self._read_test_data('asan_uaf.txt')
+    print(DATA_DIRECTORY)
```

Comment:
Delete?

Reply:
Forgot to hide my curiosity 😆

Reply:
Fixed in [e49f843](https://github.com/google/clusterfuzz/pull/2555/commits/e49f8435ca87a5f640c0abce42fbd980095bf365).